imagetest/testworkflow.go

```diff
@@ -347,6 +352,11 @@ func NewTestWorkflow(name, image, timeout string) (*TestWorkflow, error) {
 	t.wf.Name = strings.ReplaceAll(name, "_", "-")
 	t.wf.DefaultTimeout = timeout
 
+	t.osUnderTest = "linux"
+	if strings.Contains(t.Image, "windows") {
+		t.osUnderTest = "windows"
+	}
```

Comment:
imo unless there are more complex steps, i think we should just use this logic directly wherever we are currently checking `t.osUndertest`

Reply:
That's fair. Don't have a strong opinion on this. We can come back to this if for some reason OS classification becomes more complex.